src/helpers/issue.ts

```diff
@@ -206,6 +206,25 @@ export const upsertCommentToIssue = async (issue_number: number, comment: string
   }
 };
 
+export const upsertLastCommentToIssue = async (issue_number: number, commentBody: string) => {
+  const context = getBotContext();
+  const logger = getLogger();
+  const payload = context.payload as Payload;
+
+  try {
+    const comments = await context.octokit.issues.listComments({
+      owner: payload.repository.owner.login,
+      repo: payload.repository.name,
+      issue_number: issue_number,
+      per_page: 100,
+    });
```

Comment:
Looks like it doesn't fetch all the comments. please use the `getAllIssueComments` method in src/helpers/issue.ts.

Reply:
@0xcodercrane thank you for the quick review, I am on it.